XenAdmin/Wizards/GenericPages/SelectMultipleVMDestinationPage.cs

```diff
@@ -350,7 +354,8 @@ private void RestoreGridHomeServerSelectionFromMapping()
 
                     List<IEnableableXenObjectComboBoxItem> list =
                         cbCell.Items.OfType<IEnableableXenObjectComboBoxItem>().ToList();
-                    IEnableableXenObjectComboBoxItem item =
+
+                    var item =

```

Comment:
I think this could be written in one line without the intermediate list as `var item  = cbCell.Items.OfType<IEnableableXenObjectComboBoxItem>().FirstOrDefault(cbi => MatchingWithXenRefObject(cbi, mapping.XenRef));`